build/compose.yml

```diff
@@ -1,4 +1,4 @@
-name: habitat-node
+version: "3.3"
```

Comment:
where did name go?

Reply:
This field is no longer supported in more recent versions of docker: https://stackoverflow.com/questions/73427249/name-does-not-match-any-of-the-regexes-x-while-trying-to-set-the-project